packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Delete all Stay signed in sessions
+---
+
+<ApiLifecycle access="ie" />
+
+This guide explains how to delete all sessions of the **Stay signed in** feature using the [My Account Sessions API](https://developer.okta.com/docs/api/openapi/okta-myaccount/myaccount/tag/Sessions/). See [Stay signed in](https://help.okta.com/okta_help.htm?type=oie&id=ext-stay-signed-in) for more details.
+
+> **Note:** This document is written for Okta Identity Engine. If you’re using Classic Engine, time to upgrade. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
```

Comment:
```suggestion
> **Note:** This document is written for Okta Identity Engine. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
```

Reply:
I really like the conversational "time to upgrade" line but I'm not sure it'd work for translation purposes.